job-server-extras/src/main/scala/spark/jobserver/context/SessionContextFactory.scala

```diff
@@ -0,0 +1,34 @@
+package spark.jobserver.context
+
+import com.typesafe.config.Config
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.SparkSession
+import spark.jobserver.{ContextLike, SparkSessionJob}
+import spark.jobserver.api.SparkJobBase
+import spark.jobserver.util.SparkJobUtils
+
+case class SparkSessionContextLikeWrapper(spark: SparkSession) extends ContextLike {
```

Comment:
@velvia I tried that approach as my first try but it did not work as only object SparkSession is exposed and could not extend it with ContextLike trait